R/resource_update.R

```diff
@@ -97,8 +102,9 @@
 #' (xxx <- resource_update(xx, path=newpath))
 #' browseURL(xxx$url)
 #' }
-resource_update <- function(id, path, url = get_default_url(),
-  key = get_default_key(), as = 'list', ...) {
+resource_update <- function(id, path, extras,
```

Comment:
optional parameters have to have a default value (you can avoid that by using `missing()` internally, but I don't like that coding pattern) - i imagine the sensical default value would be `list()`?

Reply:
thanks for catching that